.env.example

```diff
@@ -24,10 +25,11 @@ FEATURE_TOGGLE_CONFIG='{"url":null,"apiKey":null,"static":{}}'
 BANK_ACCOUNT_REQUEST_EMAIL_TARGET=help@doma.ai
 
 # FIREBASE_CONFIG_JSON contents can be retrieved from https://console.firebase.google.com/project/<PROJECT_ID>/settings/serviceaccounts/adminsdk
+# nosemgrep: generic.secrets.security.detected-google-gcm-service-account.detected-google-gcm-service-account
 # FIREBASE_CONFIG_JSON='{"type": "service_account", "project_id": "<PROJECT_ID>", "private_key_id": "<PRIVATE_KEY_ID", "private_key": "<PRIVATE_KEY>", "client_email": "<FIREBASE_ADMIN_SDK_EMAIL> at .gserviceaccount.com", "client_id": "<CLIENT_ID>", "auth_uri": "https://accounts.google.com/o/oauth2/auth", "token_uri": "https://oauth2.googleapis.com/token", "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs", "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/<CLIENT_EMAIL_VALUE>"}'
 
 # real Firebase push token for testing real push notification delivery to device, would be used only if provided
-# FIREBASE_PUSH_TOKEN_TEST=flt0weSOS8eROf6OEZAmHp:APA91bG0I-QBvGjCL5jblzDoOuVV6bZ1x4dyRmyPqr2iZBYyFbDJcqtKw0vvzo4MH-PaLiVQJbLfAGCoryYXykdqCKXdx2QqYAk6vE4pmxrKk6RHe33-mVNHNTB0HxYI9KUsb21CHfPp
```

Comment:
Should we save this token somewhere?

Reply:
I don't know the source of this token, but anyway it should be removed from the public repo